test-utils/src/dev/clickhouse.rs

```diff
@@ -294,10 +301,14 @@ struct ClickHouseDataDir {
 }
 
 impl ClickHouseDataDir {
-    fn new() -> Result<Self, anyhow::Error> {
-        // Keepers do not allow a dot in the beginning of the directory, so we must
-        // use a prefix.
-        let dir = Utf8TempDir::with_prefix("clickhouse-")
+    fn new(logctx: &LogContext) -> Result<Self, anyhow::Error> {
+        // TODO: replace with log_prefix_for_test once
+        // https://github.com/oxidecomputer/dropshot/pull/878 is merged.
```

Comment:
This is done, right?

Reply:
Yeah, thought I removed this comment. Huh